vhdl_lang/src/syntax/configuration.rs

```diff
@@ -375,7 +377,8 @@ end;
                     block_spec: code.s1("rtl(0)").name(),
                     use_clauses: vec![],
                     items: vec![],
-                }
+                },
+                source_range: source_range(&code, (0, 0), (3, 4))
```

Comment:
Using hard coded line numbers and colums makes the tests fragile and harder to maintain. I have avoided it so far by using test helper to compute source ranges from textual searches such as finding the nth substring. In this case it would be nice to create a test helper that finds a source range between two substrings instead of hardcoding it.

Reply:
Makes sense.